include/fastdds/rtps/common/InstanceHandle.h

```diff
@@ -27,101 +29,115 @@ namespace eprosima {
 namespace fastrtps {
 namespace rtps {
 
+using KeyHash_t = std::array<octet, 16>;
+
+struct RTPS_DllAPI InstanceHandleValue_t
+{
+    octet& operator [] (
+            size_t i) noexcept
+    {
+        has_been_set_ = true;
+        return value_[i];
+    }
+
+    octet operator [] (
+            size_t i) const noexcept
+    {
+        return value_[i];
+    }
+
+    operator octet* () noexcept
+    {
+        has_been_set_ = true;
```

Comment:
Also comment this code like the suggestion in `operator[]`.

Reply:
Done in caa6bf8